srcpkgs/qbe/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'qbe'
+pkgname=qbe
+version=1.1
+revision=1
+archs="x86_64* aarch64*"
```

Comment:
Does qbe not support other architectures?

Reply:
It *can* be built on other archs (armv7, i686, etc.) but it only compiles to x86_64, aarch64, and riscv.

Reply:
>  but it only targets x86_64, aarch64, and riscv.

If by this you mean that it can compile C code to just those architectures, then restricting `archs` here is unnecessary (since from what you say qbe can be built to other archs just fine).